pkg/gardenlet/apis/config/v1alpha1/defaults_test.go

```diff
@@ -183,6 +183,7 @@ var _ = Describe("Defaults", func() {
 			Expect(obj.SyncPeriod).To(PointTo(Equal(metav1.Duration{Duration: 1 * time.Hour})))
 			Expect(obj.WaitSyncPeriod).To(PointTo(Equal(metav1.Duration{Duration: 15 * time.Second})))
 			Expect(obj.SyncJitterPeriod).To(PointTo(Equal(metav1.Duration{Duration: 5 * time.Minute})))
+			Expect(obj.JitterUpdates).To(PointTo(Equal(false)))
```

Comment:
```suggestion
			Expect(obj.JitterUpdates).To(PointTo(BeFalse()))
```

Reply:
Addressed in [commit](https://github.com/gardener/gardener/pull/5483/commits/de284d6ecaa02c1411dda2dee6accfbb949d97f5)